popdat/src/import_census.rs

```diff
@@ -1,11 +1,95 @@
-use map_model::Map;
-
 use crate::CensusArea;
+use abstutil::Timer;
+use geo::algorithm::intersects::Intersects;
+use geojson::GeoJson;
+use map_model::Map;
+use std::convert::TryFrom;
 
 impl CensusArea {
-    pub fn find_data_for_map(_map: &Map) -> Result<Vec<CensusArea>, String> {
-        // TODO importer/src/utils.rs has a download() helper that we could copy here. (And later
-        // dedupe, after deciding how this crate will integrate with the importer)
-        todo!()
+    pub fn find_data_for_map(map: &Map, timer: &mut Timer) -> Result<Vec<CensusArea>, String> {
+        // TODO eventually it'd be nice to lazily download the info needed. For now we expect a
+        // prepared geojson file to exist in data/system/<city>/population_areas.geojson
+        //
+        // When we implement downloading, importer/src/utils.rs has a download() helper that we
+        // could copy here. (And later dedupe, after deciding how this crate will integrate with
+        // the importer)
+        let path = abstutil::path(format!(
+            "system/{}/population_areas.geojson",
+            map.get_name().city
+        ));
+        let bytes = abstutil::slurp_file(&path)?;
+        debug!("parsing geojson at path: {}", &path);
+
+        // TODO - can we change to Result<_,Box<dyn std::error::Error>> and avoid all these map_err?
```

Comment:
My instincts would be to change it all to `Box<dyn Error>`, but maybe there's a reason you prefer String errors. If so, do you have any advice on making this less verbose?

I'd love to be able to write:
```
let str = String::from_utf8(bytes)?;
timer.start("parsing geojson");
let geojson = str.parse::<GeoJson>()?;
```

Have you worked with `anyhow` at all? It might be a nice middle ground in all these places where we return String errors, which aren't really meant to be "handled" other than just logged.

Reply:
I should say, I don't have much experience with `anyhow`... just some reading and conversations with other people who are happy with it for "non-graceful" error handling (and it's sister crate this_error for graceful library error handling)

Reply:
Woops, missed this thread. https://crates.io/crates/anyhow looks nice, pretty much what we need! I've avoided the Rust error ecosystem because there's _so_ much churn, but this one seems simple and popular. I'd be fine moving everything over to this.